nexus/types/src/deployment.rs

```diff
@@ -187,6 +190,7 @@ pub struct Blueprint {
     pub clickhouse_cluster_config: Option<ClickhouseClusterConfig>,
 
     /// when this blueprint was generated (for debugging)
+    #[diffus(ignore)]
```

Comment:
Do we also want to ignore `creator` and `comment`? I think we don't care about this in diffs either.

Reply:
After discussion with Rain, I'm going to write a visitor. I think it's worth allowing the possibility to diff these via visitor implementations.